cmd/ctl/pkg/install/install.go

```diff
@@ -229,12 +230,11 @@ func (o *InstallOptions) runInstall(ctx context.Context) (*release.Release, erro
 	// Install chart
 	o.client.DryRun = false     // Apply DryRun cli flags
 	o.client.ClientOnly = false // Perform install against cluster
-	// 'Atomic=True' means that if part of the install fails, all resource installs are reverted;
-	// Helm supports 3 diffent combinations of the (Atomic, Wait) boolean couple:
-	// (False, False), (False, True) or (True, True)
-	// For simplicity, we want do not support Waiting without the Atomic option (False, True),
-	// this allows this cli to use a single --wait=(True|False) flag
```

Comment:
I found this comment useful, but I also find the new, shorter comment, below, easier to read.
But what neither comment explains is why we don't also use Atomic=True, in the async install scenario (`--wait=False`).
That's my question....why don't we always set `Atomic=True`? Please answer that in a comment above the code, for future reference.